autonomous_vehicle:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 10
  front_right_wheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: frontRightWheelJoint
  front_left_wheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: frontLeftWheelJoint
  rear_right_wheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: rearRightWheelJoint
  rear_left_wheel_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: rearLeftWheelJoint
